windows.md

```diff
@@ -0,0 +1,30 @@
+Windows Node Support
+======
+
+Collecting data about windows nodes is supported by Kubecost as of v1.93.0; however, we cannot deploy kubecost or data collection daemonsets to windows nodes.
```

Comment:
> however, we cannot deploy kubecost or data collection daemonsets to windows nodes.
 
Feels like this point can be moved below and replace with what this document does, i.e. gives an overview of how to deploy, limitations, etc.
Let's capitalize Kubecost

Reply:
K, capitlized and moved below into the two sections.